designs/administrator/future-openshift/bulk-actions/index.md

```diff
@@ -0,0 +1,146 @@
+---
+parent: Administrator
+version: 4.x
+---
+
+# Bulk actions
+
+Bulk actions will be added to a number of list views in the OpenShift console. This feature will allow users to perform an action on multiple resources at once. The list of actions include adding labels, adding annotations and deleting resources.
+
+## Bulk actions in toolbar
+The bulk selector will be the first component in the toolbar on pages where bulk actions are available.
+![toolbar](img/toolbar.png)
+
+Users can click the checkbox in the bulk selector to select all items in the list (max 200 items can be selected). If all items are selected, the bulk selector checkbox will be filled with a check and the bulk selector will read ‘200 selected’, informing users that the max number of items in the list are selected. Clicking the bulk selector checkbox again will deselect all items.
```

Comment:
(max 200 items can be selected) > a maximum of 200 items can be selected at a time

Reply:
fixed!